### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by DynaMight1124** [dsolonenko/titanius-launcher#1]
Hi

Firstly, amazing work. This is now my main launcher, I love the way it looks and the way it works. Since I got an Android handheld, I realised how much I wanted that feel of ES. All my roms are prescraped with Skraper so this just works for me.

There are a few things I noticed. Not come across any bugs yet, all runs really smooth but a few more options/features could be nice. I've been playing most of the day and came up with a few.

- Allow custom directories to be used on a per system basis if required. I think the stock ones are fine and falls inline with the usual ES setup but some may have other setups forced upon them or may not want to change their whole current setup.
- Theme option to show boxart first then fade to the video 2 seconds later 
- Allow more games to be shown on the list, currently shows 7, maybe allow options for say 5, 7 and 10. Helps with large collections.
- An additional screen which can show the additional/all the metadata from the XML. Maybe pressing 'X' to bring up a 'More Info' screen.
- 12 hour time, either a toggle or just taken from the Android system preferences. 
- On my RP3+, both L1/R1 and L2/R2 move between systems. Maybe better that L1/R1 quick scrolls through the game list, either say 10 at a time or by letter. It can currently take a long time to scroll down large game lists.

I'll keep playing and add anything else.

Thanks

**Comment by dsolonenko** [comment]
Hello, thank you for the suggestions, I will look into some of them. For now, I have fixed the last point, it was on my list.

**Comment by DynaMight1124** [comment]
> Hello, thank you for the suggestions, I will look into some of them. For now, I have fixed the last point, it was on my list.

Thank you. The fast scroll works, although I feel that allowing a fast continuous scroll when the button is held might be good too, else it takes quite a bit of bashing to get to the end of a long game list. Also may even a loop back to the start when you get to the end?

**Comment by dsolonenko** [comment]
Yeah I agree, this needs to be improved

**Comment by dsolonenko** [comment]
> Theme option to show boxart first then fade to the video 2 seconds later

In my opinion, it might not be necessary to wait for 2 seconds before displaying the game video. Considering that modern Android devices can load local videos in a much shorter time, it may feel "laggy".

**Comment by DynaMight1124** [comment]
> > Theme option to show boxart first then fade to the video 2 seconds later
> 
> In my opinion, it might not be necessary to wait for 2 seconds before displaying the game video. Considering that modern Android devices can load local videos in a much shorter time, it may feel "laggy".

It was something that happened on a few of my ES devices, I quite liked it. It showed the boxart first so you knew which game it was at a glance and then showed the video if you stayed on the game with a nice fade between boxart and video. Its possible they did it due to the slowness of linux/ES loading videos but I actually quite liked it, it was almost like the best of both worlds, makes use of both boxart and video, rather than just one. 

Its totally your call, I just thought it could be a nice feature with a toggle for the end user to choose between boxart, video or both.

Just wanna say, I'm loving each new version. Been testing it as much as I can and its working great. Thank you for creating it.

**Comment by dsolonenko** [comment]
Thanks, mate, appreciate the support. Let me give it a try adding that as an option.

**Comment by jldgomes78** [comment]
Hello there!

I won't have anything useful to add other than saying THANK YOU!

It looks like you have answered my prayers, I tried every launcher for Android under the sun and the one that came closer to that emulation station "feel" was launchbox (And the only one with decent scarpping), however, it's a little bloated and slow-ish on the rp3+ (However, totally usable and good looking).

I think I finally found "home" :)
